db/migrate/20210911024734_create_portfolios.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePortfolios < ActiveRecord::Migration[6.0]
+  def change
+    create_table :portfolios do |t|
+      t.decimal :price
+      t.integer :qty
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
I think you're missing some important FK here

Reply:
mga foreign key maam, pinag iisipan pa namin mabuti mga foreign key na ilalagay po,
gawan ko nalang po yun ng bagong branch